models/person_reid_youtureid/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.24.0)
+project(opencv_zoo_person_reid_youtureid)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)

```

Comment:
Could you release standard?

OpenCV 4.x stays on C++11, so does the demo.

Reply:
We have now updated the code to use C++11 standards.